lib/p2p/Parser.ts

```diff
@@ -60,20 +60,30 @@ interface Parser {
 
 /** Protocol packet parser */
 class Parser extends EventEmitter {
+  private pending = '';
```

Comment:
I raised a similar comment earlier but I'd prefer calling this `buffer` or maybe `pendingPacket`. Actually, since it might contain multiple packets I don't think `pendingPacket` would be too good, but I still like `buffer`.

Reply:
I didn't want `buffer` to be confused with the `Buffer` type. But I can use it if you prefer, since `feed` will receive a string anyway.

Reply:
I didn't want `buffer` to be confused with the `Buffer` type. But I can use it if you prefer, since `feed` will receive a string now.